README.md

```diff
@@ -1 +1,4 @@
-# Dummy Transformations
+# Pythontex Tools
+[![Tests](https://github.com/IslasGECI/pythontex_tools/actions/workflows/actions.yml/badge.svg)](https://github.com/IslasGECI/pythontex_tools/actions/workflows/actions.yml)
+
+Pythontex Tools is a module to write the description of our data.
```

Comment:
Qué te parece esta descripción en el README? 
No estoy segura de la gramática jajaja pero lo que quiero es ampliar un poco la descripción de este repo.
```suggestion
Pythontex Tools is a module to get information from our data (such as the description or the title of our database)  and write it into a report.
```

Reply:
@MaritzaB @nepito 
Hola!
Tengo esta propuesta,

> Pythontex Tools is a module to facilitate the interaction between python and latex.
> With Pythontex Tools we can write titles, links and descriptions from our data directly to Latex.

Reply:
Me gusta. Qué te parece si (siguiendo la emoción de @nepito ) le ponemos un ejemplo de uso, algo como lo siguiente:

```suggestion
Pythontex Tools is a module to write the description of our data.

## Ejemplo de uso
Para usar este módulo debemos importarlo al archivo de latex en el que estemos trabajando nuestro reporte.

```
Y ahí le vamos poniendo el ejemplo siguiente:

```
\begin{pycode}

import pythontex_tools as ptt
data_information = '../data/raw/nidos_busqueda_aves_marinas/datapackage.json'

search_nests_datapackage = '../data/raw/nidos_busqueda_aves_marinas/datapackage.json'

metadata_writer = ptt.Writer_Metadata()
metadata_writer.load_metadata(search_nests_datapackage)

\end{pycode}
```

Reply:
excelentísimo, entonces quedaria asi:

# Pythontex Tools
[![Tests](https://github.com/IslasGECI/pythontex_tools/actions/workflows/actions.yml/badge.svg)](https://github.com/IslasGECI/pythontex_tools/actions/workflows/actions.yml)

Pythontex Tools is a module to facilitate the interaction between python and latex.
With Pythontex Tools we can write titles, links and descriptions from our data directly to Latex.

## Ejemplo de uso
Para usar este módulo debemos importarlo al archivo de latex en el que estemos trabajando nuestro reporte.

``` 
\begin{pycode}

import pythontex_tools as ptt
data_information = '../data/raw/nidos_busqueda_aves_marinas/datapackage.json'

search_nests_datapackage = '../data/raw/nidos_busqueda_aves_marinas/datapackage.json'

metadata_writer = ptt.Writer_Metadata()
metadata_writer.load_metadata(search_nests_datapackage)

\end{pycode}
```

como lo ven?
@nepito @MaritzaB

Reply:
Se ve bien, pero creo que la sección ejemplo de uso se vería mejor en inglés 🏴󠁧󠁢󠁥󠁮󠁧󠁿 jejeje

Reply:
por supuesto,
consignare esa corrección
gracias!

Reply:
Listo!

Reply:
Creo :thinking: que nos sobra la variable `data_information` creo que con la de `search_nests_datapackage` es fuciciente. Hasta ahí no hemos hecho nada, solo cargar los metadatos. [Aquí](https://bitbucket.org/IslasGECI/isla-guadalupe/pull-requests/11#Lreports/mapa_densidad_censo.texT68) y [aquí](https://bitbucket.org/IslasGECI/isla-guadalupe/pull-requests/11#Lreports/mapa_densidad_censo.texT70) está el uso de sus dos métodos públicos (a parte del `load_metadata`). 

¿@xwalls por favor podrías por favor esos dos ejemplos para acompletarlo?

Reply:
claro que si nepo, yo me encargo

Reply:
que tal asi @nepito ?

## Example of use
To use this module is needed to import it inside our LaTeX reports that we are working on.

### To import
```
\begin{pycode}

import pythontex_tools as ptt

search_nests_datapackage = '../data/raw/nidos_busqueda_aves_marinas/datapackage.json'

metadata_writer = ptt.Writer_Metadata()
metadata_writer.load_metadata(search_nests_datapackage)

\end{pycode}
```
### To use
```
\py{metadata_writer.write_title_with_link()}. \\
```
or
```
\py{metadata_writer.description()}. \\
```

Reply:
Se ve a todo dar